code/modules/vehicles/vehicle.dm

```diff
@@ -0,0 +1,328 @@
+/*
+This is the base file for the Artic Station Vehicle code,
+Dont touch this file unless you know your shit about this.
+*/
```

Comment:
For this, the original purpose was for a planet-conversion of SS13, currently on hiatus and i might aswell port it here to work on it